README.md

```diff
@@ -118,6 +118,21 @@ The normal Serverless deploy procedure will automatically bundle with Webpack:
 - Install Serverless Webpack as above
 - Deploy with `serverless deploy`
 
+### Use with serverless-offline
```

Comment:
I will change it accordingly and also add the `--location` hint (as long as I'm allowed to do that). Otherwise I open a separate PR with the same changes (as they are trivial) and close this one to get 2.0.0 out later.